package.json

```diff
@@ -0,0 +1,126 @@
+    "chrome-trace-event": "^1.0.3",
+    "clone-deep": "^4.0.1",
+    "colorette": "^2.0.19",
+    "commander": "^2.20.3",
+    "cross-spawn": "^7.0.3",
+    "cssesc": "^3.0.0",
+    "electron-to-chromium": "^1.4.194",
+    "enhanced-resolve": "^5.10.0",
+    "envinfo": "^7.8.1",
+    "es-module-lexer": "^0.9.3",
+    "escalade": "^3.1.1",
+    "eslint-scope": "^5.1.1",
+    "esrecurse": "^4.3.0",
+    "estraverse": "^4.3.0",
+    "events": "^3.3.0",
+    "fast-deep-equal": "^3.1.3",
+    "fast-json-stable-stringify": "^2.1.0",
+    "fastest-levenshtein": "^1.0.14",
+    "find-up": "^4.1.0",
+    "function-bind": "^1.1.1",
+    "glob-to-regexp": "^0.4.1",
+    "graceful-fs": "^4.2.10",
+    "has": "^1.0.3",
+    "has-flag": "^4.0.0",
+    "icss-utils": "^5.1.0",
+    "import-local": "^3.1.0",
+    "interpret": "^2.2.0",
+    "is-core-module": "^2.9.0",
+    "is-plain-object": "^2.0.4",
+    "isexe": "^2.0.0",
+    "isobject": "^3.0.1",
+    "jest-worker": "^27.5.1",
+    "json-parse-even-better-errors": "^2.3.1",
+    "json-schema-traverse": "^0.4.1",
+    "kind-of": "^6.0.3",
+    "loader-runner": "^4.3.0",
+    "locate-path": "^5.0.0",
+    "lru-cache": "^6.0.0",
+    "merge-stream": "^2.0.0",
+    "mime-db": "^1.52.0",
+    "mime-types": "^2.1.35",
+    "nanoid": "^3.3.4",
+    "neo-async": "^2.6.2",
+    "node-releases": "^2.0.6",
+    "p-limit": "^2.3.0",
+    "p-locate": "^4.1.0",
+    "p-try": "^2.2.0",
+    "path-exists": "^4.0.0",
+    "path-key": "^3.1.1",
+    "path-parse": "^1.0.7",
+    "picocolors": "^1.0.0",
+    "pkg-dir": "^4.2.0",
+    "postcss": "^8.4.14",
+    "postcss-modules-extract-imports": "^3.0.0",
+    "postcss-modules-local-by-default": "^4.0.0",
+    "postcss-modules-scope": "^3.0.0",
+    "postcss-modules-values": "^4.0.0",
+    "postcss-selector-parser": "^6.0.10",
+    "postcss-value-parser": "^4.2.0",
+    "punycode": "^2.1.1",
+    "randombytes": "^2.1.0",
+    "rechoir": "^0.7.1",
+    "resolve": "^1.22.1",
+    "resolve-cwd": "^3.0.0",
+    "resolve-from": "^5.0.0",
+    "safe-buffer": "^5.2.1",
+    "schema-utils": "^3.1.1",
+    "semver": "^7.3.7",
+    "serialize-javascript": "^6.0.0",
+    "shallow-clone": "^3.0.1",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "source-map": "^0.6.1",
+    "source-map-js": "^1.0.2",
+    "source-map-support": "^0.5.21",
+    "supports-color": "^8.1.1",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "tapable": "^2.2.1",
+    "terser": "^5.14.2",
+    "terser-webpack-plugin": "^5.3.3",
+    "update-browserslist-db": "^1.0.5",
+    "uri-js": "^4.4.1",
+    "util-deprecate": "^1.0.2",
+    "watchpack": "^2.4.0",
+    "webpack-merge": "^5.8.0",
+    "webpack-sources": "^3.2.3",
+    "which": "^2.0.2",
+    "wildcard": "^2.0.0",
+    "yallist": "^4.0.0"
+  },
+  "devDependencies": {
+    "css-loader": "^6.7.1",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.73.0",
+    "webpack-cli": "^4.10.0"
```

Comment:
- [ ] Kindly be noticed that you to add also the `webpack-dev-server` package as devDependencies in `packages.json`.